ios/Podfile.lock

```diff
@@ -1,223 +1,4 @@
 PODS:
```

Comment:
this is how many pods are removed by just pre-building the firestore pod 😱

Reply:
wow :000